tests/unit/test_bot_handlers.py

```
@@ -1,243 +1,251 @@
+    mock_llm.assert_called_once()
+
+
+@pytest.mark.asyncio
+async def test_handler_error_handling(mock_update_with_user, mock_context):
+    """Test error handling in handlers."""
+    # Mock an error in message.reply_text
+    mock_update_with_user.message.reply_text.side_effect = Exception("Test error")
+
+    # Should not raise exception - but our current handlers don't have error handling
+    # So we expect this to raise for now
+    with pytest.raises(Exception):
```

Ruff (0.13.1)

Do not assert blind exception: `Exception`

(B017)